Makefile

```diff
@@ -1,19 +1,27 @@
+CONTAINER_NAME=packit
+CONTAINER_EXEC=podman exec $(CONTAINER_NAME)
+
 prepare-check:
 	ansible-playbook -b -K -i inventory-local -c local ./recipe-tests.yaml
 
-check:
-	tox
+container:
+	rsync -a $(CURDIR)/ /tmp/packit
+	podman container inspect packit || podman run --name $(CONTAINER_NAME) -ti -d -v /tmp/packit:/src:Z -w /src fedora:30
```

Comment:
Why not `-v $(CURDIR):/src:Z`?

Reply:
because CentOS-CI :-) 
everything runs under ``/root`` dir and selinux does not allow to use ``:Z`` for changing content inside ``/root`` dir and without ``:Z`` it causes permission denied. so that I have to copy git content to some another location and then I can use it.